src/components/status__cookie.pug

```diff
@@ -0,0 +1,3 @@
+.status--banner.theme--caution.cookie#cookie
+  .container
+    p Parliament.uk use cookies to make the site simpler. #[+link('#', 'Find out more about cookies.')]
```

Comment:
Should this link to the /meta/cookie-policy page?

Reply:
Not necessary to add links here